API.rst

```diff
@@ -0,0 +1,161 @@
+The Skulpt Web Application API
+==============================
+
+This is a proposal for a new API to Skulpt, if adopted and implemented it will replace the current hodgepodge of functions currently used to embed Skulpt in an HTML web page.
+
+A few key preliminary points
+----------------------------
+
+* A smallish but more nicely and consistently named set of functions could wrap the current stuff and make life easier for people trying to get started.
+* The old functions would remain so as to provide backward compatibility, in fact I anticipate that this new api would largely be a thin wrapper over the old.  So keeping the old is important for backward compatibiliy and for advanced users who may want more fine grained control.
+
+Objects and Methods
+-------------------
+
+* skulpt.exec(code) -- handles promises, exceptions, calls skulpt.error when an error occurs...
+* skulpt.eval(code)  -- returns a javascript object resulting from evaling a Python expression
```

Comment:
I guess I had envisioned a true expression evaluator (knowing that we cannot do it right now) returning a remaped result.